modules/log6x-log-collector-syslog-server.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log-collector-syslog-server_{context}"]
+= Configuring the collector to listen for connections as a syslog server
+
+You can configure your log collector to collect journal format infrastructure logs by specifying `syslog` as a receiver input in the `ClusterLogForwarder` custom resource (CR).
+
+:feature-name: Syslog receiver input
+include::snippets/logging-http-sys-input-support.adoc[]
+
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {clo}.
+* You have created a `ClusterLogForwarder` CR.
+
+.Procedure
+
+. Grant the `collect-infrastructure-logs` cluster roles to the service account:
```

Comment:
```suggestion
. Grant the `collect-infrastructure-logs` cluster role to the service account by running the following command:
```

- Should it be "role"? if not ignore this comment.
- Add "by running the...command:"

Reply:
should be role, good catch. Updated